src/Sylius/Bundle/CoreBundle/Fixture/ChannelFixture.php

```diff
@@ -41,6 +41,7 @@ protected function configureResourceNode(ArrayNodeDefinition $resourceNode): voi
                 ->booleanNode('enabled')->end()
                 ->booleanNode('skipping_shipping_step_allowed')->end()
                 ->booleanNode('skipping_payment_step_allowed')->end()
+                ->booleanNode('account_verification_required')->defaultTrue()->end()
```

Comment:
We handle the default values in `OptionsResolver` in the example factory, could you remove `->defaultTrue()` here and customise `configureOptions` method in `ChannelExampleFactory`?